kqueen/blueprints/api/generic_views.py

```diff
@@ -111,6 +125,13 @@ def dispatch_request(self, *args, **kwargs):
 
         return jsonify({'id': self.obj.id, 'state': 'deleted'})
 
+    def get_policy_key(self):
```

Comment:
I would move this method override here - https://github.com/Mirantis/kqueen/blob/9e6955ab850cc7a9bd5b90e8a1932d01fe5420e1/kqueen/blueprints/api/views.py#L414 - you can then skip the if condition, because it will always be 'user:delete'